### PR TITLE
changes sherpa spec files for pyext , scons and swig to work

### PR DESCRIPTION
**Pull request by iahmad-khan**
https://github.com/cms-sw/cmsdist/issues/2341

### CODE REVIEW

**Comment by iahmad-khan** [comment]
please test

File: sherpa.spec [hunk 1]
```diff
@@ -53,3 +54,14 @@ make %{makeprocesses}
...
+
+%post
+%{relocateConfig}lib/python2.7/site-packages/ufo_interface/sconstruct_template
+%{relocateConfig}bin/make2scons
+%{relocateConfig}share/SHERPA-MC/makelibs
+%{relocateConfig}bin/Sherpa-config
+%{relocateConfig}bin/Sherpa-generate-model
+%{relocateConfig}include/SHERPA-MC/ATOOLS/Org/CXXFLAGS.H
+
+
```
Comment (davidlt) [inline sherpa.spec]: Remove empty lines.

File: sherpa.spec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 %define branch cms/v%realversion
 %define github_user cms-externals
 Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}-%{realversion}&output=/%{n}-%{realversion}-%{tag}.tgz
-Requires: hepmc lhapdf blackhat sqlite fastjet openssl
+Requires: hepmc lhapdf blackhat sqlite fastjet openssl scons swig
```
Comment (davidlt) [inline sherpa.spec]: Do we actually need SWIG as run-time dependency or SWIG is only build-time dependency?

File: sherpa-toolfile.spec [hunk 3]
```diff
@@ -30,6 +30,8 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/sherpa.xml
 <use name="fastjet"/>
 <use name="sqlite"/>
 <use name="openloops"/>
+<use name="scons"/>
+<use name="swig"/>
```
Comment (davidlt) [inline sherpa-toolfile.spec]: Verify that SWIG is needed at run-time (e.g. something links to SWIG libraries)

**Comment by iahmad-khan** [comment]
please test

**Comment by iahmad-khan** [comment]
transient error 'urllib2.HTTPError: HTTP Error 503: Service Unavailable'

**Comment by iahmad-khan** [comment]
please test

**Comment by iahmad-khan** [comment]
please test

**Comment by iahmad-khan** [comment]
please test

**Comment by iahmad-khan** [comment]
please test

**Comment by iahmad-khan** [comment]
@smuzaffar should we merge it

**Comment by smuzaffar** [comment]
looks good

**Pull request merged**